lib/internal/streams/promises.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const {
+  Promise,
+} = primordials;
+
+function pipeline() {
+  return new Promise((resolve, reject) => {
+    require('internal/streams/pipeline')(...arguments, (err, value) => {
```

Comment:
Please move the requires to the top.

Reply:
Maybe that's okay? https://github.com/nodejs/node/pull/33991#discussion_r443198867 https://github.com/nodejs/node/pull/33991#discussion_r443198919